packages/checkout/widgets-lib/src/widgets/connect/components/WalletList.tsx

```diff
@@ -261,6 +262,23 @@ export function WalletList(props: WalletListProps) {
     setShowWalletDrawer(true);
   }, [track]);
 
+  useEffect(() => {
+    // Auto-trigger wallet connection via rdns
+    if (targetWalletRdns && targetWalletRdns?.length > 0) {
+      if (targetWalletRdns === WalletProviderRdns.PASSPORT && passportProviderDetail) {
+        handleWalletItemClick(passportProviderDetail);
+      } if (targetWalletRdns === WalletProviderRdns.WALLETCONNECT && walletConnectItemRef.current) {
```

Comment:
@dreamoftrees please run prettier, this formatting is off

Reply:
Can we huddle when you have a sec please @jhesgodi .. my eslint and prettier are not happy..